core/client/ui/form-log-in.js

```diff
@@ -39,6 +39,9 @@ const passwordlessLogin = template => {
     if (err && err.message !== 'User not found [403]') {
       lp.notif.error(err.message);
       return;
+    } else if (err && err.message === 'User not found [403]') {
```

Comment:
I don't know if we really need to indicate this when an account doesn't exist. Indicating that an account does not exist allows a user to know if a person has an account on the platform, which is not great in terms of privacy

Reply:
I agree with that (and that's why I did it that way originally) but the pain here is that it is hard for a user to know if something is wrong with the email (not sent, in the wrong mailbox, whatever), or if his account doesn't exist.
It's a bit annoying in term of UX.

There is already a way to know if an account exists, by trying to create a new one with the same email (I agree that it is not very stealthy, though ^^).

@Donorhan Do you prefer a configurable setting on the error message privacy or do we accept it as a drawback of passwordless login?